src/@orb.yml

```diff
@@ -3,10 +3,9 @@ version: 2.1
 description: >
   Test and deploy APEX applications to Salesforce.
 
-# This information will be displayed in the orb registry and is not mandatory.
 display:
-  home_url: "https://www.website.com/docs"
-  source_url: "https://www.github.com/EXAMPLE_ORG/EXAMPLE_PROJECT"
+  source_url: "https://github.com/CircleCI-Public/salesforce-apex-orb/"
```

Comment:
Add the salesforce home url. I would recommend using https://developer.salesforce.com/